src/thumbnailer_slope_optim.cc

```diff
@@ -27,13 +27,26 @@ Thumbnailer::Status Thumbnailer::GenerateAnimationSlopeOptim(
   CHECK_THUMBNAILER_STATUS(LossyEncodeSlopeOptim(webp_data));
   CHECK_THUMBNAILER_STATUS(NearLosslessEqual(webp_data));
 
+  // If there is no frame encoded in near-lossless, use
+  // GenerateAnimationEqualQuality() to generate animation.
+  int num_near_ll_frames = 0;
+  for (auto& frame : frames_)
```

Comment:
Please use curling braces for the `for` loop here. They may be omitted for the `if` block, however at WebP we only skip them if both the condition and its statement fit on the same line, such as:
```c++
for (auto& frame : frames_) {
  if (frame.near_lossless) ++num_near_ll_frames;
}
```
https://google.github.io/styleguide/cppguide.html#Loops_and_Switch_Statements

Also if the type is not very long, try to avoid `auto`. Here the `frame` can be `const` also.